app/components/verkiezingen/draft-mandataris-list.js

```diff
@@ -22,20 +22,16 @@ export default class DraftMandatarisListComponent extends Component {
   }
 
   get resortedMandatarissen() {
-    if (!this.args.sort || this.args.sort.indexOf('rangorde') < 0) {
-      return this.mandatarissen;
+    if (!this.mandatarissen) {
```

Comment:
Could also give it a start value when defining the tracked property but i think its better to just check it here if it is set so we do not expect default behaviour of variables.